source/features/linkify-branch-refs.tsx

```diff
@@ -32,7 +32,7 @@ function inPR() {
 		}
 	}
 
-	for (const el of select.all('.commit-ref')) {
+	for (const el of select.all<HTMLElement>('.commit-ref')) {
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />`, but not certain.

Reply:
I'd stick to `HTMLElement`